crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -444,6 +444,8 @@ en:
       identify: 'Identify'
     form:
       allocate: 'Allocate'
+      allocate_node: 'Allocate Node'
+      confirm_allocate: 'WARNING! Allocating the node will wipe all the disks attached to the node including the shared disks. Proceed?'
```

Comment:
If you ask `Proceed?` you would expect `Ok` and `Cancel` . instead it says `Allocate`, so maybe drop the `Proceed?`